nvflare/fuel/f3/stats_pool.py

```diff
@@ -33,17 +40,36 @@ def _check_name(cls, name, scope):
                 return name
         raise ValueError(f"pool '{name}' is already defined")
 
+    @classmethod
+    def set_hist_pool_config(cls, config: dict):
+        if not isinstance(config, dict):
+            raise ValueError(f"config data must be dict but got {type(config)}")
+        for k, v in config.items():
+            cls.hist_pool_config[k.lower()] = v
```

Comment:
One minor thing here, I know we only use this "stats_pool_config" right now for hist_pool only.
But we could potentially use it for other type of pools.

Maybe we change this method name to more general "set_stats_pool_config"?
and the class variable rename from hist_pool_config to stats_pool_config